app.rb

```diff
@@ -72,16 +74,16 @@ def create_rental
       puts 'What date the book was rented? (dd/mm/yyyy)'
       date = gets.chomp
       (book, person) = find_bookperson(title, id)
-      Rental.new(date: date, person: Person[person], book: Book[book])
+      @rentals_array.push(Rental.new(date: date, book: book, person: person))
       puts 'Rental added successfully'
     end
   end
 
   def rentals_list
-    if Rental.size
+    if @rentals_array.empty?
       puts 'No existing rentals'
     else
-      Rental.each do |rental|
+      @rentals_array.each do |rental|
```

Comment:
[OPTIONAL] You did a great job implementing all previous reviewer requests. However, if I try to display the rentals, it does not show me the book's name and person. Kindly, fix it.